### ISSUE TITLE
Results for VOC2012 are not correct 

### ISSUE THREAD

**Issue by daicoolb** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#51]
I Use the original images from VOC2012 for training.
training set: 1464  
test set :  1449
At the same time , I use the Res101 for pre-train model which is from slim's checkpoint.
The loss for training is about 1.8
when I test the model, the IOU is very low , only 0.1 and some classes are nan,
What's wrong with my configuration ?  I have modified the environment with README

**Comment by daicoolb** [comment]
Restored model parameters from model_0827/model.ckpt-20000
step 0
step 100
step 200
step 300
step 400
step 500
step 600
step 700
step 800
step 900
step 1000
step 1100
step 1200
step 1300
step 1400
Pixel Accuracy: 0.969
Mean IoU: 0.462
class 0: 0.000
/app/home/deeplab/model.py:338: RuntimeWarning: divide by zero encountered in long_scalars
  IoU = TP / (TP + FP + FN)
class 1: 0.000
class 2: 0.000
class 3: 0.000
class 4: 0.000
class 5: 0.000
class 6: 0.000
class 7: 0.000
class 8: 0.000
class 9: 0.000
class 10: 0.000
class 11: 0.000
class 12: 0.000
class 13: 0.000
class 14: 0.000
class 15: 0.000
class 16: 0.000
class 17: 0.000
class 18: 0.000
class 19: 0.000
class 20: 0.000
mIoU: 0.000

**Comment by zhengyang-wang** [comment]
"RuntimeWarning: divide by zero encountered in long_scalars"

Obviously, there is something wrong with your label images.

**Comment by zhengyang-wang** [comment]
First, check that your label images are composed of pixels with integer values between [0,20].
Second, as I used augmented VOC2012 dataset, I don't know whether there are some crucial differences between the original dataset and the augmented one, other than more training images. You may want to check it.

One thing that I'm pretty sure is that using ResNet-101 slim checkpoint is fine and will not cause problems.
